zhaquirks/sonoff/snzb02d.py

```diff
@@ -0,0 +1,111 @@
+"""Sonoff SNZB-02D - Zigbee LCD smart temperature humidity sensor."""
+
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import QuirkBuilder
+from zigpy.quirks.v2.homeassistant import PERCENTAGE, UnitOfTemperature
+import zigpy.types as t
+from zigpy.zcl.foundation import BaseAttributeDefs, DataTypeId, ZCLAttributeDef
+
+
+class TemperatureUnit(t.enum16):
+    """Temperature unit."""
+
+    Celsius = 0
+    Fahrenheit = 1
+
+
+class CustomSonoffCluster(CustomCluster):
+    """Sonoff custom cluster."""
+
+    cluster_id = 0xFC11
+
+    class AttributeDefs(BaseAttributeDefs):
+        """Attribute definitions."""
+
+        comfort_temperature_max = ZCLAttributeDef(
+            id=0x0003,
+            type=t.int16s,
+        )
+
+        comfort_temperature_min = ZCLAttributeDef(
+            id=0x0004,
+            type=t.int16s,
+        )
+
+        comfort_humidity_min = ZCLAttributeDef(
+            id=0x0005,
+            type=t.uint16_t,
+        )
+
+        comfort_humidity_max = ZCLAttributeDef(
+            id=0x0006,
+            type=t.uint16_t,
+        )
+
+        temperature_unit = ZCLAttributeDef(
+            id=0x0007,
+            type=TemperatureUnit,
+            zcl_type=DataTypeId.uint16,
+        )
+
+    @property
+    def _is_manuf_specific(self):
+        return False

```

Comment:
I think using the `manufacturer_id_override` for this class should also work instead of overriding `_is_manuf_specific` now:
```python
    # remove manufacturer id for cluster
    manufacturer_id_override: t.uint16_t = foundation.ZCLHeader.NO_MANUFACTURER_ID
```

with the import at the top:
```python
from zigpy.zcl import foundation
```